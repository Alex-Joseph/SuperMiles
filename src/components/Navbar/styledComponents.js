import styled from 'styled-components';


const StyledNavbar = styled.div`
  height: 50px;
  display: grid;
  align-items: center;
  padding: 0 8px;
  font-size: 24px;
  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.3);
`;

export { StyledNavbar };
