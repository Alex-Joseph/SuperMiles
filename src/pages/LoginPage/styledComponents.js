import styled from 'styled-components';


const StyledLoginWrapper = styled.div`
  display: grid;
  place-content: center;
  padding: 24px 12px;
  grid-template-columns: 350px;
`;

export { StyledLoginWrapper };
