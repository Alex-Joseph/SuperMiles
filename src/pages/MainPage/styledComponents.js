import styled from 'styled-components';


const StyledMarketPlaceWrapper = styled.div`
  display: grid;
  place-content: center;
  padding: 24px 12px;

  h1 {
    text-align: center;
  }
`;

const StyledProductsWrapper = styled.div`
  display: grid;
  grid-template-columns: 1fr 1fr;

  div {
    display: grid;
    justify-items: center;
  }
`;

export { StyledMarketPlaceWrapper, StyledProductsWrapper };
