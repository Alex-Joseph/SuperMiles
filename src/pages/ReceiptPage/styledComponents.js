import styled from 'styled-components';


const StyledMarketPlaceWrapper = styled.div`
  display: grid;
  place-content: center;
  padding: 24px 12px;

  h1 {
    text-align: center;
  }
`;

export { StyledMarketPlaceWrapper };
