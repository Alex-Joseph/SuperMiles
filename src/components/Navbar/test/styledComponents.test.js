import React from 'react';
import { shallow } from 'enzyme';
import { StyledNavbar } from '../styledComponents';


describe('StyledNavbar', () => {
  it('renders correct snapshot for a StyledNavbar', () => {
    const wrapper = shallow(<StyledNavbar />);

    expect(wrapper).toMatchSnapshot();
  });
});
