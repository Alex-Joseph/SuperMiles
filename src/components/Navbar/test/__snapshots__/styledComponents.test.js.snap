// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledNavbar renders correct snapshot for a StyledNavbar 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  height: 50px;
  display: grid;
  align-items: center;
  padding: 0 8px;
  font-size: 24px;
  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.3);
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
